src/App.js

```diff
@@ -1,5 +1,29 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
```

Comment:
`MissionUtils` 모듈은 해당 미션에서 사용되는 `Console` 외에도 다른 함수들을 포함해요.
이 미션에서는 `Console`만 사용하기때문에 `Console` 만 로드하는 것을 추천해요

```suggestion
import { Console } from "@woowacourse/mission-utils";
```